ui-app/client/src/components/SmallCarousel/smallCarouselStyleProperties.ts

```diff
@@ -0,0 +1,735 @@
+import { StyleGroupDefinition, StylePropertyDefinition } from '../../types/common';
+
+export const styleProperties: Array<StylePropertyDefinition> = [
+	{
+		groupName: 'Font',
+		displayName: 'Slider Font',
+		name: 'sliderFont',
+		defaultValue: '<primaryFont>',
+		cssProperty: 'font',
+		selector: '.comp.compSmallCarousel',
+		noPrefix: true,
+	},
+	{
+		groupName: 'Text Color',
+		displayName: 'Slider Font Color',
+		name: 'sliderFontColor',
+		defaultValue: '<fontColorOne>',
+		cssProperty: 'color',
+		selector: '.comp.compSmallCarousel',
+		noPrefix: true,
+	},
+
+	{
+		groupName: 'Design One',
+		displayName: ' Slider ContainerBackground ColorDesign One',
+		name: ' SliderContainerBackgroundColorDesignOne',
```

Comment:
Rework on the names for the variables and remove the color scheme column in the excel sheet.